src/pathwayvis/services/actions/actions.service.ts

```diff
@@ -65,6 +66,7 @@ class Knockout extends ReactionAction implements Action {
 
 
   public canDisplay(context) {
+    console.log("KNOCKOUT");
```

Comment:
it's fine to leave comments while the PR is work in progress (WIP), but preferable use brakpoints